modules/ipfs-cpinner-service/src/api.ts

```diff
@@ -14,7 +14,7 @@ export function setupPublicApi (app: Express, provider: IpfsPinnerProvider, logg
 
     const content = await provider.get(did, key)
 
-    res.status(200).json({ content })
+    res.status(200).json(content)
```

Comment:
Why?

Reply:
Before:
```
content: [
  'this is a content',
  'this is another content'
]
```
By returning everything under `content` we are specifying that it is an array of contents

Now:
```
[
  { id: '123456789', content: 'this is a content' },
  { id: '987654321', content: 'this is another content' }
]
```
Now we are specifying the meaning inside the object

Reply:
Brutal change, ok! Let's document it wisely.